extensions/bitfinex/README.md

```diff
@@ -0,0 +1,13 @@
+# Bitfinex Extension
+
+## Configuration
+
+1. Go to [Bitfinex API keys](https://setting.bitfinex.com/api#my-keys) and create a new API key with the following permission
+   - **Account History**: `Get historical balances entries and trade information`
+   - **Orders**: `Create and cancel orders`
+   - **Margin Funding** permissions
+   - **Wallets**: `Get wallet balances and addresses
+    ![API Permission](media/api_permission.png)
+2. Open the extension command and paste your API key and secret into the fields.
+3. Choose the currency you want to use.
+   ![Extension Preference](media/preference.png)
```

Comment:
I would remove the images from the README, they are not properly aligned with the content and I think a good text description in this case is already good enough.

Reply:
okay! 👍